metadata.json

```diff
@@ -7,7 +7,5 @@
     "version_minor": 0,
     "api_major": 3,
     "api_minor": 0,
-    "categories": "AudioVideo;Audio;",
-    "home_url": "https://www.amazon.com/gp/dmusic/cloudplayer/player",
-    "allowed_uri" : "^https?://www.amazon.com/gp/dmusic/cloudplayer/player#?.*$"
```

Comment:
Should this be removed entirely or just modified to support the different top-level domains?
Possibly `^https?://www.amazon.[a-z.]+/gp/dmusic/cloudplayer/player#?.*$`